src/components/dialogs/GifSelect.tsx

```diff
@@ -45,12 +45,12 @@ export function GifSelectDialog({
   let snapPoints
   switch (externalEmbedsPrefs?.giphy) {
     case 'show':
-      content = <GifList control={control} onSelectGif={onSelectGif} />
+      content = <GifList onSelectGif={onSelectGif} />
```

Comment:
No changes in this file, just making some adjustments to make sure we use the `close(() => onClose())` pattern to avoid breaking things.